pkg/queue/stats/stats_reporter.go

```diff
@@ -0,0 +1,190 @@
+/*
+Copyright 2019 Google Inc. All Rights Reserved.
```

Comment:
Wrong Copyright.

Reply:
Done. I'll fix where I copied it from :)